Userland/Libraries/LibJS/Runtime/ArrayBuffer.cpp

```diff
@@ -27,7 +32,13 @@ ArrayBuffer::ArrayBuffer(size_t byte_size, Object& prototype)
 
 ArrayBuffer::ArrayBuffer(ByteBuffer& buffer, Object& prototype)
     : Object(prototype)
-    , m_buffer(buffer)
+    , m_buffer(ByteBuffer { buffer })
```

Comment:
As discussed on Discord earlier - this can probably just be

```suggestion
    , m_buffer(buffer)
```